pipelines/broad/genotyping/illumina/example.json

```diff
@@ -6,18 +6,18 @@
   "IlluminaGenotypingArray.reported_gender": "Female",
   "IlluminaGenotypingArray.chip_well_barcode": "7991775143_R01C01",
 
-  "IlluminaGenotypingArray.green_idat_cloud_path": "gs://broad-gotc-test-storage/arrays/HumanExome-12v1-1_A/idats/7991775143_R01C01/7991775143_R01C01_Grn.idat",
-  "IlluminaGenotypingArray.red_idat_cloud_path": "gs://broad-gotc-test-storage/arrays/HumanExome-12v1-1_A/idats/7991775143_R01C01/7991775143_R01C01_Red.idat",
+  "IlluminaGenotypingArray.green_idat_cloud_path": "gs://broad-public-datasets/IlluminaGenotypingArrays/inputs/HumanExome-12v1-1_A/idats/7991775143_R01C01_Grn.idat",
+  "IlluminaGenotypingArray.red_idat_cloud_path": "gs://broad-public-datasets/IlluminaGenotypingArrays/inputs/HumanExome-12v1-1_A/idats/7991775143_R01C01_Red.idat",
 
-  "IlluminaGenotypingArray.bead_pool_manifest_file": "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExome-12v1-1_A.bpm",
-  "IlluminaGenotypingArray.extended_chip_manifest_file": "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExome-12v1-1_A.1.3.extended.csv",
-  "IlluminaGenotypingArray.cluster_file": "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExomev1_1_CEPH_A.egt",
-  "IlluminaGenotypingArray.gender_cluster_file": "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/HumanExomev1_1_gender.egt",
-  "IlluminaGenotypingArray.zcall_thresholds_file": "gs://broad-gotc-test-storage/arrays/metadata/HumanExome-12v1-1_A/IBDPRISM_EX.egt.thresholds.txt",
+  "IlluminaGenotypingArray.bead_pool_manifest_file": "gs://broad-public-datasets/IlluminaGenotypingArrays/metadata/HumanExome-12v1-1_A/HumanExome-12v1-1_A.bpm",
+  "IlluminaGenotypingArray.extended_chip_manifest_file": "gs://broad-public-datasets/IlluminaGenotypingArrays/metadata/HumanExome-12v1-1_A/HumanExome-12v1-1_A.1.3.extended.csv",
+  "IlluminaGenotypingArray.cluster_file": "gs://broad-public-datasets/IlluminaGenotypingArrays/metadata/HumanExome-12v1-1_A/HumanExomev1_1_CEPH_A.egt",
+  "IlluminaGenotypingArray.gender_cluster_file": "gs://broad-public-datasets/IlluminaGenotypingArrays/metadata/HumanExome-12v1-1_A/HumanExomev1_1_gender.egt",
```

Comment:
Can you remove this line ('gender_cluster_file' and the next 'zcall_thresholds_file' entirely) and verify that the json still works (it should).

Those two inputs are for more advanced kind of Broad-specific features that other users probably won't use.

Reply:
Done